src/opencl_common.c

```diff
@@ -1163,7 +1163,7 @@ static char *get_build_opts(int sequential_id, const char *opts)
 #else
 	        (options.verbosity >= VERB_MAX &&
 	         gpu_nvidia(device_info[sequential_id])) ?
-	         "-cl-nv-verbose " : "",
+	         "-cl-nv-verbose -cl-std=CL1.2 " : "",
```

Comment:
This doesn't work - except when verbosity is bumped (obvious if you look closer).